sdk/aqueduct/flow.py

```diff
@@ -153,3 +154,19 @@ def describe(self) -> None:
             )
         )
         print(json.dumps(self.list_runs(), sort_keys=False, indent=4))
+
+    def get(self, artifact_name: str) -> Artifact:
+        """Gets the Artifact from the flow based on artifact_name"""
+        resp = self._api_client.get_workflow(self._id)
+        assert (
+            len(resp.workflow_dag_results) > 0
+        ), "Every flow must have at least one run attached to it."
+        latest_result = resp.workflow_dag_results[-1]
```

Comment:
Are we assuming here that when we have a handle to a `flow`, it always points to the latest version @kenxu95?

Reply:
I guess I'm really asking how this interacts with versioning -- can we access older versions?

Reply:
yeah, @Fanjia-Yan  this method shouldn't be on the flow, this should be on the flow run. A flow itself is actually dag-agnostic right now, since a dag is always associated with a particular run. You can also grab a run with `flow.latest()` or `flow.fetch()`